### PR TITLE
Content token shuffle

### PR DESCRIPTION
**Pull request by benjaminleonard**
Implementation of https://github.com/oxidecomputer/design-system/pull/93

image

`content-secondary` -> `content-default` and now becomes the standard text colour.

Old `content-default` dimmed slightly and becomes `content-raise` and used in more places.

### CODE REVIEW

File: app/ui/lib/Badge.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ export const badgeColors: Record<BadgeVariant, Record<BadgeColor, string>> = {
     default: `ring-1 ring-inset bg-accent-secondary text-accent ring-[rgba(var(--base-green-800-rgb),0.15)]`,
     destructive: `ring-1 ring-inset bg-destructive-secondary text-destructive ring-[rgba(var(--base-red-800-rgb),0.15)]`,
     notice: `ring-1 ring-inset bg-notice-secondary text-notice ring-[rgba(var(--base-yellow-800-rgb),0.15)]`,
-    neutral: `ring-1 ring-inset bg-secondary text-secondary ring-[rgba(var(--base-neutral-700-rgb),0.15)]`,
+    neutral: `ring-1 ring-inset bg-secondary text-default ring-[rgba(var(--base-neutral-700-rgb),0.15)]`,
```
Comment (david-crespo) [inline app/ui/lib/Badge.tsx:31]: neutral badge has lost its background color, not sure what that's about

image

File: app/components/DocsPopover.tsx [hunk 2]
```diff
@@ -45,22 +45,22 @@ export const DocsPopover = ({ heading, icon, summary, links }: DocsPopoverProps)
   return (
     <Popover>
       <PopoverButton className={cn(buttonStyle({ size: 'sm', variant: 'ghost' }), 'w-8')}>
-        <Info16Icon aria-label="Links to docs" className="shrink-0" />
+        <Info16Icon aria-label="Links to docs" className="shrink-0 text-tertiary" />
```
Comment (david-crespo) [inline app/components/DocsPopover.tsx:48]: tertiary looks kinda sad next to the brighter refresh button and the stop button. it also looks kinda sad on its own

image

image

**Comment by david-crespo** [comment]
Very into it overall, good to tone down the side modal form header and other places the bright white was punching you in the face a little

File: app/ui/lib/Badge.tsx (hunk 1 above)
Comment (benjaminleonard) [reply app/ui/lib/Badge.tsx:31]: Fixed, it's because `bg-inverse-tertiary` no longer exists.

File: app/components/DocsPopover.tsx (hunk 2 above)
Comment (benjaminleonard) [reply app/components/DocsPopover.tsx]: Brightened the whole lot up

File: app/ui/lib/Table.tsx [hunk 3]
```diff
@@ -115,7 +115,7 @@ Table.Cell = ({ height = 'small', className, children, ...props }: TableCellProp
  * Used _outside_ of the `Table`, this element wraps buttons that sit on top
  * of the table.
  */
-export const TableActions = classed.div`-mt-11 mb-3 flex justify-end gap-2`
+export const TableActions = classed.div`-mt-6 mb-3 flex justify-end gap-2`
```
Comment (david-crespo) [inline app/ui/lib/Table.tsx:118]: I don't like it lol. but it'll look a lot better when the table actions row is more filled out

Comment (benjaminleonard) [reply app/ui/lib/Table.tsx:118]: Was weird to have it shuffle on different pages – instances and projects.

**Pull request merged**